blacktie/pom.xml

```diff
@@ -177,7 +177,7 @@
             <dependency>
                 <groupId>org.jboss.narayana.jts</groupId>
                 <artifactId>narayana-jts-jacorb</artifactId>
-                <version>${project.version}</version>
+                <version>5.3.3.Final</version>
```

Comment:
It's hard coded to the version in WFLY 10.1.0 so we don't lose track of the dependency. Otherwise we could build with APIs that won't be there at runtime for BT